app/controllers/users_controller.rb

```diff
@@ -13,7 +13,7 @@ def index
     @users = User.joins(:maps)
                  .select('users.*, count(users.id) as maps_count')
                  .group('maps.user_id')
-                 .order(sort_column + ' ' + sort_direction)
+                 .order(sort_column => sort_direction)
```

Comment:
Ah, could we do `sort_column: sort_direction` here? 

Then, let's rebase and this looks great! Thank you @StlMaris123 !!!